WalletWasabi.Fluent/ViewModels/Wallets/Home/History/Details/CoinJoinDetailsViewModel.cs

```diff
@@ -55,6 +57,8 @@ private void Update()
 			Confirmations = transaction.Confirmations;
 			IsConfirmed = Confirmations > 0;
 			TransactionId = transaction.Id;
+			ConfirmationTime = _wallet.Transactions.TryEstimateConfirmationTime(transaction.Id);
+			IsConfirmationTimeVisible = ConfirmationTime.HasValue && ConfirmationTime != TimeSpan.Zero;
```

Comment:
`ConfirmationTime` and `IsConfirmationTimeVisible` is a simple property, will never fire NotifyPropertyChanged event.
You should modify them to have `[AutoNotify]` attribute.

Reply:
So true! I forgot to modify this class as well.